collect_app/src/main/java/org/odk/collect/android/tasks/InstanceServerUploaderTask.java

```diff
@@ -80,7 +77,7 @@ public Outcome doInBackground(Long... instanceIdsToUpload) {
                 outcome.messagesByInstanceId.put(instance.getDbId().toString(),
                         customMessage != null ? customMessage : getLocalizedString(Collect.getInstance(), R.string.success));
 
-                analytics.logEvent(SUBMISSION, "HTTP", Collect.getFormIdentifierHash(instance.getFormId(), instance.getFormVersion()));
+                Analytics.log(SUBMISSION, "HTTP", Collect.getFormIdentifierHash(instance.getFormId(), instance.getFormVersion()));
```

Comment:
This can also become a "form" event assuming we're dropping Google Drive events.